Mathlib/LinearAlgebra/FiniteDimensional/Defs.lean

```diff
@@ -694,21 +693,6 @@ noncomputable def fieldOfFiniteDimensional (F K : Type*) [Field F] [h : CommRing
   { divisionRingOfFiniteDimensional F K with
     toCommRing := h }
 end
-
-namespace Submodule
-
-section DivisionRing
-
-variable [DivisionRing K] [AddCommGroup V] [Module K V] {V₂ : Type v'} [AddCommGroup V₂]
-  [Module K V₂]
-
-theorem finrank_mono [FiniteDimensional K V] : Monotone fun s : Submodule K V => finrank K s :=
```

Comment:
This is subsumed and generalised by the new lemma called `Submodule.finrank_mono`. Can you mention this in the commit message, please? (Or restore this lemma, appropriately named?)

Reply:
Mentioned in the PR description